TestFoundation/TestURLResponse.swift

```diff
@@ -11,56 +11,63 @@ class TestURLResponse : XCTestCase {
-        let mimetype = "APPlication/wordperFECT"
-        let res = URLResponse(url: URL(string: "test")!, mimeType: mimetype, expectedContentLength: 0, textEncodingName: nil)
+        
+        mimetype = "APPlication/wordperFECT"
+        res = URLResponse(url: testURL, mimeType: mimetype, expectedContentLength: 0, textEncodingName: nil)
+        XCTAssertEqual(res.mimeType, mimetype, "should be the other mimetype")
+        
+        mimetype = nil
+        res = URLResponse(url: testURL, mimeType: mimetype, expectedContentLength: 0, textEncodingName: nil)
         XCTAssertEqual(res.mimeType, mimetype, "should be the other mimetype")
     }
-
+    
     func test_ExpectedContentLength() {
-        let zeroContentLength = 0
-        let positiveContentLength = 100
-        let url = URL(string: "test")!
-        let res1 = URLResponse(url: url, mimeType: "text/plain", expectedContentLength: zeroContentLength, textEncodingName: nil)
-        XCTAssertEqual(res1.expectedContentLength, Int64(zeroContentLength), "should be Int65 of the zero length")
-        let res2 = URLResponse(url: url, mimeType: "text/plain", expectedContentLength: positiveContentLength, textEncodingName: nil)
-        XCTAssertEqual(res2.expectedContentLength, Int64(positiveContentLength), "should be Int64 of the positive content length")
+        var contentLength = 100
+        var res = URLResponse(url: testURL, mimeType: "text/plain", expectedContentLength: contentLength, textEncodingName: nil)
+        XCTAssertEqual(res.expectedContentLength, Int64(contentLength), "should be positive Int64 content length")
+        
+        contentLength = 0
+        res = URLResponse(url: testURL, mimeType: nil, expectedContentLength: contentLength, textEncodingName: nil)
+        XCTAssertEqual(res.expectedContentLength, Int64(contentLength), "should be zero Int64 content length")
+        
+        contentLength = -1
+        res = URLResponse(url: testURL, mimeType: nil, expectedContentLength: contentLength, textEncodingName: nil)
+        XCTAssertEqual(res.expectedContentLength, Int64(contentLength), "should be invalid (-1) Int64 content length")
     }
     
     func test_TextEncodingName() {
-        let encoding = "utf8"
-        let url = URL(string: "test")!
-        let res1 = URLResponse(url: url, mimeType: nil, expectedContentLength: 0, textEncodingName: encoding)
-        XCTAssertEqual(res1.textEncodingName, encoding, "should be the utf8 encoding")
-        let res2 = URLResponse(url: url, mimeType: nil, expectedContentLength: 0, textEncodingName: nil)
-        XCTAssertNil(res2.textEncodingName)
+        var encoding = "utf8"
+        var res = URLResponse(url: testURL, mimeType: nil, expectedContentLength: 0, textEncodingName: encoding)
+        XCTAssertEqual(res.textEncodingName, encoding, "should be the utf8 encoding")
+        
+        let res = URLResponse(url: testURL, mimeType: nil, expectedContentLength: 0, textEncodingName: nil)
```

Comment:
This did not compile:

> /home/buildnode/jenkins/workspace/swift-corelibs-foundation-PR-Linux/swift-corelibs-foundation/TestFoundation/TestURLResponse.swift:66:13: error: definition conflicts with previous value

Reply:
Very sorry to waste your time, this should not have happened 🙇‍♂️ I have setup Mojave now and can directly develop and test on the master branch.

It's fixed, please ci-re-test? 🙏